fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -1973,6 +2016,42 @@ public long getConnectorSinkTargetMaxFileSize() {
     @VarAttr(name = ENABLE_PARTITION_COLUMN_VALUE_ONLY_OPTIMIZATION, flag = VariableMgr.INVISIBLE)
     private boolean enablePartitionColumnValueOnlyOptimization = true;
 
+    @VarAttr(name = ENABLE_USE_ANN)
+    private boolean enableUseANN = false;
+
+    @VarAttr(name = USE_IVFPQ)
+    private boolean useIVFPQ = false;
+
+    @VarAttr(name = ANN_TYPE)
+    private String annType = ANNAlgorithmType.TYPE_DEFAULT;
+
+    @VarAttr(name = VECTOR_DISTANCE_COLUMN_NAME)
+    private String vectorDistanceColumnName = DEFAULT_VECTOR_DISTANCE_COLUMN_NAME;
+
+    @VarAttr(name = VECTOR_LIMIT_K)
+    private long vectorLimitK;
+
+    @VarAttr(name = IS_REWRITTEN_VECTOR_PLAN)
+    private boolean isRewrittenVectorPlan = false;
+
+    @VarAttr(name = QUERY_VECTOR)
+    private List<String> queryVector = new ArrayList<>();
+
+    @VarAttr(name = VECTOR_RANGE)
+    private double vectorRange = -1;
+
+    @VarAttr(name = RESULT_ORDER)
+    private int resultOrder = 0;
+
+    @VarAttr(name = QUERY_PARAMS)
+    private Map<String, String> queryParams = Maps.newHashMap();
+
+    @VarAttr(name = PQ_REFINE_FACTOR)
+    private double pqRefineFactor = 1;
+
+    @VarAttr(name = K_FACTOR)
+    private double kFactor = 1;
```

Comment:
better introduce a common prefix for those variables, some of the names need more clearity.